<?xml version="1.0"?>
<launch>
    <arg name="output" default="screen"/>
    <arg name="frame" default="map"/>
    <rosparam file="$(find robotFace)/config/butia_emotions_controllers.yaml" command="load"/>

    <node pkg="robotFace" type="eyelidController.py" name="eyelidController" output="$(arg output)" />
    <node pkg="robotFace" type="eyebrownController.py" name="eyebrowController" output="$(arg output)" />
    <node pkg="robotFace" type="emotionController.py" name="emotionController" output="$(arg output)" />
    <node pkg="robotFace" type="newNeckController.py" name="newNeckController" output="$(arg output)">
        <arg name="frame" value="$(arg frame)"/>
    </node>
    <node pkg="robotFace" type="dataController.py" name="dataController" output="$(arg output)" />
</launch>
